setup.py

```diff
@@ -68,7 +66,7 @@ def is_requirement(line):
     license='AGPL',
     url='https://github.com/edx/pytest-warnings-report',
     description='A pytest plugin for generating warnings reports.',
-    long_description=read('README.rst'),
+    long_description=README,
```

Comment:
have you tested the output? why do we need to remove `read` here?

Reply:
because it is already read and stored in README variable https://github.com/edx/pytest-warnings-report/blob/9981d9bd484abdb6feca72d2db68abd007b738d1/setup.py#L55